ports/randungeon/gameinfo.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="utf-8"?>
+<gameList>
+  <game>
+    <path>./Afterlife Gladiator.sh</path>
+    <name>Afterlife Gladiator</name>
+    <desc> A fantasy turn based RPG about dead people fighting each other.</desc>
+    <developer>BakarLabs</developer>
+    <publisher>BakarLabs</publisher>
+    <genre>rpg</genre>
+    <image>./afterlifegladiator/screenshot.jpg</image>
+  </game>
+</gameList>
```

Comment:
gameinfo.xml is written for a game called Afterlife Gladiator instead of Randungeon.